source/mail-access.rst

```diff
@@ -10,10 +10,12 @@ Webmail
 You can access your emails using the webmail interface at `webmail.uberspace.de 
 <https://webmail.uberspace.de>`_. Use your full email address and the corresponding password to login. This works with your ``@uber.space`` address as well as with any addresses using your :ref:`own domains <mail-domains>`.
 
-For your ``@uber.space`` address the password is the same as for SSH access, which you can set in your `dashboard
-<https://dashboard.uberspace.de/dashboard/authentication>`_.
+SystemMail address access
```

Comment:
The structure is _how_ to access your mails, alternatives are webmail and client. This is not a how but a what, I don't think this should be another headline.